node/wipe_appwrite_bucket/src/index.js

```diff
@@ -0,0 +1,103 @@
+const sdk = require("node-appwrite");
+
+/*
+  'req' variable has:
+    'headers' - object with request headers
+    'payload' - request body data as a string
+    'variables' - object with function variables
+
+  'res' variable has:
+    'send(text, status)' - function to return text response. Status code defaults to 200
+    'json(obj, status)' - function to return JSON response. Status code defaults to 200
+
+  If an error is thrown, a response with code 500 will be returned.
+*/
+
+module.exports = async function (req, res) {
+  const client = new sdk.Client();
+
+  const storage = new sdk.Storage(client);
+
+  console.log("Starting function");
+
+  // initialize appwrite client
+  if (
+    !req.variables["APPWRITE_FUNCTION_ENDPOINT"] ||
+    !req.variables["APPWRITE_FUNCTION_API_KEY"]
+  ) {
+    console.warn(
+      "Environment variables are not set. Function cannot use Appwrite SDK."
+    );
+    res.json({ success: false, message: "Environment variables are not set." });
+    return;
+  } else {
+    client
+      .setEndpoint(req.variables["APPWRITE_FUNCTION_ENDPOINT"])
+      .setProject(req.variables["APPWRITE_FUNCTION_PROJECT_ID"])
+      .setKey(req.variables["APPWRITE_FUNCTION_API_KEY"]);
+  }
+ 
+  // parse payload
+  const payload = req.payload;
+  const { bucketId } = JSON.parse(payload);
+  if (!bucketId) {
+    console.warn("Bucket ID is not set.");
+    res.json({ success: false, message: "Bucket ID is not set." });
+    return;
+  }
+
+  // get files
+  let filesFound = true;
+  let offset = 0;
+  let files = [];
+
+  console.log("Getting files from bucket: " + bucketId);
+  while (filesFound) {
+    try {
+      // fetch files from appwrite
+      const sto = await storage.listFiles(bucketId, [
+        sdk.Query.limit(2),
```

Comment:
I think we can handle a bigger limit than 2.

Reply:
LMAO yeah I forgot to change that after debugging 😂